bin/rebuildShrinkwrap.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+/**
+ * This script will install the package and version you want in the directories
+ * that are defined in below array. It accept a full version string
+ * e.g. @jenkins-cd/design-language@0.0.105-TimePrecise
+ * as input and if you do not provide that it will start a prompt.
+ *
+ * We will prune and install BEFORE we install the requested version to make sure that
+ * shrinkwrap will update correctly everytime. We further do an optional `mvn install`
+ * to publish the new hpi to the local .m2 repository
+ *
+ * Usage: bin/cleanInstall.js [[@scope/]package@version] [dev] [mvn]
+ *
+ * dev will install a devDependency
+ * mvn will run the optional `mvn install` afterwards
+ */
```

Comment:
Is this meant to replace `bin/cleanInstall.js` ?

Reply:
No, i need to fix up that file